database/users/db.go

```diff
@@ -24,6 +24,11 @@ const (
          						                	WHERE plugin_id = '86DC3AE7-9F3D-42CB-85A3-A71ABC3C3CB8'
          						                    AND attribute_name = 'wallet'
          						                    AND value->'wallet' ? $1);`
+	getWalletByUserID = `SELECT value -> 'wallet' ->> 0 AS wallet
```

Comment:
In some cases a user has two string values stored in the wallet array, are you sure you are grabbing the right one?
i.e:
`{"wallet": ["5Fn7h2HS8kx5Rf8LKkztpNHxd2wuyTmX316XpB2bbKsok7GB", "0xa449fe48eb456f4353d695e0cf810a088e9685a8216f97d56e452680cd13af26"]}`